cl/simple_pages/templates/help/relative_dates_help.html

```diff
@@ -0,0 +1,75 @@
+{% extends "base.html" %}
+
+{% block title %}Help with Relative Dates – CourtListener.com{% endblock %}
```

Comment:
Let's do this in various places: 

```suggestion
{% block title %}Help with Relative Date Queries – CourtListener.com{% endblock %}
```

Reply:
Done.